beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -125,19 +125,54 @@ func (s *InitialSync) roundRobinSync(genesis time.Time) error {
 		sort.Slice(blocks, func(i, j int) bool {
 			return blocks[i].Slot < blocks[j].Slot
 		})
+		var checkParentExists func(blk *eth.BeaconBlock, parentBlocks []*eth.BeaconBlock) ([]*eth.BeaconBlock, error)
```

Comment:
why is this like this ? make this another pointer receiver method

Reply:
changed